src/community/geopkg/src/main/java/org/geoserver/geopkg/wps/GeneralizedTablesExtension.java

```diff
@@ -51,7 +51,8 @@ public GeoPkgExtension getExtension(String name, GeoPackage geoPackage) {
         }
 
         @Override
-        public GeoPkgExtension getExtension(Class extensionClass, GeoPackage geoPackage) {
+        public GeoPkgExtension getExtension(
+                @SuppressWarnings("rawtypes") Class extensionClass, GeoPackage geoPackage) {
```

Comment:
Wondering if GeoPkgExtensionFactory should be fixed instead. Even just setting a Class<?> there.

Reply:
At `GeoPkgExtensionFactory`:
```
 public GeoPkgExtension getExtension(String name, GeoPackage geoPackage);
```
So it's ok to return `GeoPkgExtension` subclassess. Changed this one to return `GeneralizedTablesExtension`